packages/mistral/conf/mistral.conf

```diff
@@ -840,3 +843,4 @@
 
 # Enables user authentication in pecan. (boolean value)
 #auth_enable = true
+auth_enable = false
```

Comment:
Ok, found it's used for OpenStack only:
https://pypi.python.org/pypi/mistral#configuring-mistral

Merging.